pom.xml

```diff
@@ -427,13 +440,12 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
-                <version>1.0</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>enforce</goal>
                         </goals>
-                        <configuration>
+                        <configuration combine.children="append">
```

Comment:
I think both of these exclusions could be deleted, if

* any explicit `slf4j` dep were deleted (the parent POM provides it)
* the XStream dep were deleted, as above